actions/changesWebAction.js

```diff
@@ -79,7 +79,13 @@ function main(params) {
             .then(() => {
                  return createTrigger(db, triggerID, newTrigger);
             })
-            .then(resolve)
+            .then(() => {
+                 resolve({
+                     statusCode: 200,
+                     headers: {'Content-Type': 'application/json'},
+                     body: new Buffer(JSON.stringify({'status': 'success'})).toString('base64'),
```

Comment:
No, the JSON needs to be Base64 encoded and the controller decodes it when serving up the response.